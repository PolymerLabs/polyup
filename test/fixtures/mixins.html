<!DOCTYPE html>
<html>
<head>
  <title>mixins</title>
  <script src='components/webcomponentsjs/webcomponents.js'></script>
  <link rel='import' href='components/polymer/polymer.html'>
  <link rel='import' href='components/core-resizable/core-resizable.html'>
</head>
<body>

<polymer-element name='resizable-thing'>
  <script>
    Polymer(Polymer.mixin({}, Polymer.CoreResizable));
  </script>
</polymer-element>
<resizable-thing></resizable-thing>

</body>
</html>
