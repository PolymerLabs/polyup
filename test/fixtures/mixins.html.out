<!DOCTYPE html><html><head>
  <title>mixins</title>
  <script src="components/webcomponentsjs/webcomponents-lite.js"></script>
  <link rel="import" href="components/polymer/polymer.html">
  <link rel="import" href="components/iron-resizable-behavior/iron-resizable-behavior.html">
</head>
<body>

<dom-module id="resizable-thing">
</dom-module>
<script>
  Polymer({
    is: 'resizable-thing',
    behaviors: [Polymer.IronResizableBehavior]
  });
</script>
<resizable-thing></resizable-thing>



</body></html>
